package/usr/local/bin/hassbian-config

```diff
@@ -80,17 +80,35 @@ function check-permission {
 }
 
 function share-log {
+  if [ ! -f $LOGFILE ];then
+    echo "No logfile exiting..."
+    return 0
+  fi
   if [ "$ACCEPT" == "true" ]; then
-    a=$(cat $LOGFILE); curl -X POST -s -d "$a" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}'
+    loglink=$(curl -X POST -s -d "$(cat "$LOGFILE")" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}')
+    if [[ $loglink != *"initial"*  ]]; then
+      echo "$loglink"
+    else
+      echo "There is an issue with your network conection, or with the hastebin API.."
+      echo "Try again later.."
+      return 0
+    fi
   else
     echo "This will put the output from your last operation on hastebin."
     echo "This could include sensitive informastion."
     echo "If you are unsure about what it contains, you can run 'hassbian-config log' to check."
     echo -n "Do you want to create an hastebin link? [N/y] : "
     read -r  RESPONSE
     if [ "$RESPONSE" == "y" ] || [ "$RESPONSE" == "Y" ]; then
-      echo ""
-    a=$(cat $LOGFILE); curl -X POST -s -d "$a" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}'
+      loglink=$(curl -X POST -s -d "$(cat "$LOGFILE")" https://hastebin.com/documents | awk -F '"' '{print "https://hastebin.com/"$4}')
+      if [[ $loglink != *"initial"*  ]]; then
+        echo "$loglink"
+      else
+        echo
+        echo "There is an issue with your network conection, or with the hasebin API.."
```

Comment:
Spelling:

"There is an issue with your network connection, or with the Hastebin API."